### DISCUSSION TITLE
something wrong in my new proxy provider

### DISCUSSION THREAD

**Discussion by superit63** [engageub/InternetIncome#137] [Q&A]
Hello,
I am testing new proxy provider; however, I get the error. here is the logs:
![image](https://github.com/engageub/InternetIncome/assets/169600953/484961c6-23b0-4628-9284-f9dd3bfb514f)
I am configed Use_proxy=True and dont modified anythings else.
Thanks

**Comment by engageub** [comment]
Hello,
The error indicates that the server is not reachable.
There have been no code changes made recently in main branch.
Could you please recheck the proxy or the app you are using. 
Please make sure the proxies are in the correct format. 

Thank you

**Reply by superit63** [reply]
Yeah, I have checked the proxy by curl command and it works well. I am using honeygain atm. Its so weird

**Reply by superit63** [reply]
![image](https://github.com/engageub/InternetIncome/assets/169600953/5768ff12-7827-46c4-a8a4-320deb9f3f74)
Here is the logs inside Honeygain app

**Reply by engageub** [reply]
> Yeah, I have checked the proxy by curl command and it works well. I am using honeygain atm. Its so weird

Are you using webshare.io by any chance? They block the IPs.
Could you please use any other app and check the logs. 

Thank you

**Reply by engageub** [reply]
If your password has special characters, please make sure to use single quotes in properties.conf file.

Thank you

**Reply by superit63** [reply]
Nope. I didnt use webshare (I have been suspended my account @@).
![image](https://github.com/engageub/InternetIncome/assets/169600953/56945057-e457-44ef-b780-56f0cfdf3a18)
Still the same with earnapp.

**Reply by engageub** [reply]
Is this the first time are you using proxies?
Are the proxies in the correct format as mentioned in ReadMe.

Thank you

**Reply by superit63** [reply]
Thanks Engageub, I have used your Script for a month (with webshare proxy provider) so definitely, I use the right format with socks5://user:pass@ip:port
Is there any possibility can cause these error?

**Reply by engageub** [reply]
In that case, it might be your VPS or proxy issue.
You can try with other free proxies and check the logs.
Is your proxy IP authenticated or works everywhere.
Does your proxy have any special characters in the password.

Thank you

**Reply by superit63** [reply]
Hello,
You'are right, my VPS's issue. I have changed to another VPS; it works; However, I dont know why my old VPS prevent these things@@. Its so weird.
Btw, Thanks a lot for your help

**Reply by engageub** [reply]
> Hello, You'are right, my VPS's issue. I have changed to another VPS; it works; However, I dont know why my old VPS prevent these things@@. Its so weird. Btw, Thanks a lot for your help

Hello, 
Thank you for the confirmation.
Have you checked  the proxy on your old VPS using curl command.

Thank you
